docs/Makefile

```diff
@@ -11,14 +11,14 @@ SCHEMABUILD   = python $(SOURCEDIR)/dump_schema.py
 
 # Put it first so that "make" without argument is like "make help".
 help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+    @$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```

Comment:
when I tried this change, the makefile became invalid. You can test this by trying to run the doc test job. 

That's why i pinned the dependency in https://github.com/microsoft/Olive/pull/1332/files#diff-2b4945591edfeaa4cf4d3f155e66d4b43d1bda7a55d881d5cf3107f1b05abbbcR3

Reply:
Makes sense. I'll close this PR. Thanks!

Reply:
Oh, I only saw this reopened after I merged the other PR. could you undo this version pin? https://github.com/microsoft/Olive/blob/610a3b57bf3ce3fdda6ed26f24d30d8822bea3e6/requirements-dev.txt#L3

Thanks!